### DISCUSSION TITLE
Open Weather Map API keys are getting rejected.

### DISCUSSION THREAD

**Discussion by J9234450** [jeroenterheerdt/HAsmartirrigation#513] [Q&A]
Hello All 

I have generated 3 different API keys in Open Weather Map and all of them  are getting rejected.  Any thoughts on what to try next?  

Thanks, 
Jack

**Comment by jeroenterheerdt** [comment]
Wait. See documentation.

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
From: J9234450 ***@***.***>
Sent: Wednesday, December 4, 2024 11:49:49 PM
To: jeroenterheerdt/HAsmartirrigation ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [jeroenterheerdt/HAsmartirrigation] Open Weather Map API keys are getting rejected. (Discussion #513)


Hello All

I have generated 3 different API keys in Open Weather Map and all of them are getting rejected. Any thoughts on what to try next?

Thanks,
Jack
